src/carnot/exec/otel_export_sink_node.cc

```diff
@@ -49,6 +49,8 @@ const int64_t kOTelTraceIDLength = 16;
 
 const int64_t kB3ShortTraceIDLength = 8;
 
+constexpr auto otelClientTimeout = std::chrono::seconds{5};
```

Comment:
Instead of hardcoding, do you think we can pass it in with some default value through the node?